roles/matrix-bridge-appservice-webhooks/defaults/main.yml

```diff
@@ -0,0 +1,110 @@
+
+# Once you make a control room in Matrix, you can get its ID by typing any message and checking its source
+matrix_appservice_webhooks_control_room_id: ''
+matrix_appservice_webhooks_bot_name: 'webhookbot'
+matrix_appservice_webhooks_user_prefix: '_webhook'
+
+# Controls the webhooks_PORT and MATRIX_PORT of the installation
+matrix_appservice_webhooks_matrix_port: 6789
+matrix_appservice_webhooks_webhooks_port: 6788
+
+# Controls whether the appservice-webhooks container exposes its HTTP port (tcp/6788 in the container).
+#
+# Takes an "<ip>:<port>" or "<port>" value (e.g. "127.0.0.1:9999"), or empty string to not expose.
+matrix_appservice_webhooks_container_http_host_bind_port: ''
+
+matrix_appservice_webhooks_homeserver_media_url: "matrix.{{ matrix_domain }}"
+matrix_appservice_webhooks_homeserver_url: "http://matrix-synapse:8008"
+matrix_appservice_webhooks_homeserver_domain: "{{ matrix_domain }}"
+matrix_appservice_webhooks_appservice_url: 'http://matrix-appservice-webhooks'
+
+# A list of extra arguments to pass to the container
+matrix_appservice_webhooks_container_extra_arguments: []
+
+# List of systemd services that matrix-appservice-webhooks.service depends on.
+matrix_appservice_webhooks_systemd_required_services_list: ['docker.service', 'matrix-synapse.service']
+
+# List of systemd services that matrix-appservice-webhooks.service wants
+matrix_appservice_webhooks_systemd_wanted_services_list: []
+
+matrix_appservice_webhooks_appservice_token: ''
+matrix_appservice_webhooks_homeserver_token: ''
+matrix_appservice_webhooks_id_token: ''
+
+matrix_appservice_webhooks_configuration_yaml: |
+
+  # Configuration specific to the application service. All fields (unless otherwise marked) are required.
+  homeserver:
+    # The domain for the client-server API calls.
+    url: "{{ matrix_appservice_webhooks_homeserver_url }}"
+
+    # The domain part for user IDs on this home server. Usually, but not always, this is the same as the
+    # home server's URL.
+    domain: "{{ matrix_domain }}"
+
+  # Configuration specific to the bridge. All fields (unless otherwise marked) are required.
+  webhookBot:
+    # The localpart to use for the bot. May require re-registering the application service.
+    localpart: "_webhook"
+
+    # Appearance options for the Matrix bot
+    appearance:
+      displayName: "Webhook Bridge"
+      avatarUrl: "http://i.imgur.com/IDOBtEJ.png" # webhook icon
+
+  # Provisioning API options
+  provisioning:
+    # Your secret for the API. Required for all provisioning API requests.
+    secret: 'warummussesdennsolangsein'
+
+  # Configuration related to the web portion of the bridge. Handles the inbound webhooks
+  web:
+    hookUrlBase:  "{{ matrix_appservice_webhooks_inbound_uri_prefix }}"
+
+  logging:
+    file: data/webhook.log
+    console: true
+    consoleLevel: info
+    fileLevel: verbose
+    writeFiles: true
```

Comment:
We can probably consider flipping this to `false` to disable file-based logging.

Since we log to systemd/journald already and all other services avoid logging to files manually (where possible), it will be consistent with that.

I haven't tested whether this is the correct way to do it though.

Reply:
Well, if we do that, we might want to bump `consoleLevel` to `verbose`.
I'm guessing this service doesn't spew too many log entries, so verbose could be a good default for it.
I'm fine to go with your recommendation though.

Reply:
Yet again, completely right about file based logging. I flipped ```writeFiles``` to ```false``` but I will stick with ```info``` for ```consoleLevel``` since the webhook bridge actually does spill out quite a lot of information.

Reply:
In order to change the verbosity level, I added ```matrix_appservice_webhooks_log_level``` variable.

Reply:
Sounds good!